Formula/ooniprobe.rb

```diff
@@ -166,16 +166,57 @@ def install
     (var/"lib/ooni").mkpath
   end
 
-  def post_install
-    system bin/"ooniresources"
-    system bin/"oonideckgen", "-o", "#{HOMEBREW_PREFIX}/share/ooni/decks/"
```

Comment:
I presume you got Git'd in the way it display its diffs. Sometimes when you move an entire block it shows the wrong one as having been moved in a diff. Can be frustrating on things like this.